flink-core/src/test/java/org/apache/flink/types/StringSerializationTest.java

```diff
@@ -56,6 +56,27 @@ public void testNonNullValues() {
 			fail("Exception in test: " + e.getMessage());
 		}
 	}
+
+	@Test
+	public void testUnicodeValues() {
+		try {
```

Comment:
Do we need the `try-catch`? If the test case just throws Exception, it should fail and print stack trace like you implemented it.

Reply:
I just did it in the way it's done with other string serialization tests.

Reply:
Ah then leave it and we will eventually clean up on our end.